### ISSUE TITLE
Channels/Private Groups/Direct Messages disappear after beeing added to Favorites

### ISSUE THREAD

**Issue by o-jans** [RocketChat/Rocket.Chat#11147]
### Description:
Channels, Private Groups and Direct Messages disappear after beeing added to Favorites. Only after a reload they suddenly appear again.

### Steps to reproduce:

1. Go to a Channel, Private Group or Direct Message.
2. Click on the star for Favorites.

### Expected behavior:

The new favored Channel, Private Group or Direct Message should be shown instantly under Favorites. 

### Actual behavior:
The new favored Channel, Private Group or Direct Message disappears completly until a Browser refresh (or on Desktop Client a Current Server Reload) is performed.

## Additional information:
Rarely it's like excpected. But next time it is wrong again.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.65.1
- Operating System: Ubuntu
- Deployment Method: Snap
- Number of Running Instances: 1

**Comment by vynmera** [comment]
Duplicate of #10927
I believe it's fixed, but I'm not sure, try the latest `develop`?

**Comment by kaiiiiiiiii** [comment]
//CC @TwizzyDizzy It's a duplicate as @vynmera said :)

**Comment by MartinSchoeler** [comment]
Closing this since is a duplicate
